### PR TITLE
Updates to Intune iOS MAM SDK dev guide

### PR DESCRIPTION
**Pull request by Kyle-Reis**
Removing obsolete instructions on stripping out simulator architectures. Removing information on DebugSettingsEnabled flag that is not supported for 3rd party/LOB apps. Adding info on app intents.

### CODE REVIEW

File: memdocs/intune/developer/app-sdk-ios.md [hunk 1]
```diff
@@ -81,12 +81,7 @@ To enable the Intune App SDK, follow these steps:
 
     :::image type="content" source="media/app-sdk-ios/intune-app-sdk-ios-linked-framework.png" alt-text="Intune App SDK iOS Framework: Xcode Frameworks, Libraries, and Embedded Content sample":::
 
-   > [!NOTE]
-   > If you use Framework, you must manually strip out the simulator architectures from the universal framework before you submit your app to the App Store. 
-
    **Option 2 - Static Library**:  Link `libIntuneMAMSwift.xcframework` and `IntuneMAMSwiftStub.xcframework` to the target: Drag `libIntuneMAMSwift.xcframework` and `IntuneMAMSwiftStub.xcframework` to the **Frameworks, Libraries, and Embedded Content** list of the project target.  
-
-
     :::image type="content" source="media/app-sdk-ios/intune-app-sdk-ios-linked-static-lib.png" alt-text="Intune App SDK iOS Static Library: Xcode Frameworks, Libraries, and Embedded Content sample":::
```
Comment (JohanFreelancer9) [inline memdocs/intune/developer/app-sdk-ios.md:85]:
```suggestion

        :::image type="content" source="media/app-sdk-ios/intune-app-sdk-ios-linked-static-lib.png" alt-text="Intune App SDK iOS Static Library: Xcode Frameworks, Libraries, and Embedded Content sample":::
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @Kyle-Reis, kindly commit the minor suggested change. Many thanks.

**Comment by yogkumgit** [comment]
@Kyle-Reis Gentle reminder. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Erikre Please signoff this PR. Thanks

**Review by Erikre — APPROVED** [review]

**Comment by Erikre** [comment]
#sign-off

**Comment by yogkumgit** [comment]
@tiburd Please merge this Pr. Thanks

**Pull request merged**